app/static/linux/github

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Currently this file does not have the executable bit set after install:

```
$ ls -l "/opt/GitHub Desktop/resources/app/static"
total 960
...
-rw-r--r-- 1 root root    760 Mar 20 11:14 github
...
```

Should we just `chmod a+x` this file so everyone can launch it?

Reply:
ah, good catch! I might have done that without thinking or assumed it was done on the folder level on installation; yes, we'd want to make it executable